api/interfaces/interfaces.go

```diff
@@ -115,6 +115,8 @@ type User struct {
 	Password                        string               `json:"password"  bson:"password"`
 	Items                           []InventoryItem      `json:"items"     bson:"items"`
 	Loomies                         []primitive.ObjectID `json:"loomies"   bson:"loomies"`
+	ValidationCode                  string               `json:"validationCode"  bson:"validationCode"`
+	TimeExpiration                  int64                `json:"timeExpiration"   bson:"timeExpiration"`
```

Comment:
Maybe ValidationCodeExpiration or ValidationCodeExp can be more clearer.

Reply:
Are you requesting the change just for the field TimeExpiration, right? Or another field?

Reply:
Just that one.